src/main/scala/com/github/traviscrawford/spark/dynamodb/ParsedFilterExpression.scala

```diff
@@ -0,0 +1,44 @@
+package com.github.traviscrawford.spark.dynamodb
+
+/** Simple parser for filter_expression options.
+  *
+  * This will add Dynamo indirection so that reserved keywords can
+  * be handled in filter expressions.
+  *
+  * Regex matching inspired by:
+  * https://ikaisays.com/2009/04/04/using-pattern-matching-with-regular-expressions-in-scala/
+  */
+private object ParsedFilterExpression {
```

Comment:
I agree this makes sense. If in practice folks actually need something fancier, let's address at that time.